test/unit/org/apache/cassandra/db/virtual/ClientsTableTest.java

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
In {{ToolRunner}} you have methods to invoke nodetool so you could add a test for nodetool as well.

Reply:
I wanted to add more info. If you look at `ToolRunner.invokeNodetool()` callers you'll get plenty of examples of such tests. You will see there are methods to check return codes, std output, errors if any are expected, etc. `TableStatsTest` could be used as a guide. But feel free to do your take on it.

The important bit here is that test will check i.e.: The tool doesn't explode when using that option, the new info is indeed returned, return codes are as expected, tests things from the same top-level point of view a user will, etc.

Please let me know if you find any trouble getting it to work and I can happily lend a hand.